### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by Galaxias**
Updating the download composer script to work with Symfony3

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -12,8 +12,8 @@ a database, then you're in the right place.
 Prerequisites
 -------------
 
-This version of the bundle requires Symfony 2.1+. If you are using Symfony
-2.0.x, please use the 1.2.x releases of the bundle.
+This version of the bundle requires Symfony 3+. If you are using Symfony
```
Comment (1ed) [inline Resources/doc/index.rst]: This is not true. The master branch is compatible with Symfony 2.3, 2.7, 2.8, 3.0 (and master I think).

File: Resources/doc/index.rst [hunk 2]
```diff
@@ -50,7 +50,7 @@ Require the bundle with composer:
 
 .. code-block:: bash
 
-    $ composer require friendsofsymfony/user-bundle "~2.0@dev"
+    $ composer require friendsofsymfony/user-bundle "dev-master"
```
Comment (xabbuh) [inline Resources/doc/index.rst]: I wouldn't do that. This won't restrict your versions to 2.x anymore, but could pull in 3.0 dev versions (once they are started) which can lead to totally unexpected behaviour.

Comment (gruberjuliannama) [inline Resources/doc/index.rst]: Event;
use FOS\UserBundle\Model\GroupInterface;
use Symfony\Component\EventDispatcher\Event;
use Symfony\Component\HttpFoundation\Request;
class GroupEvent extends Event
{
    private $group;
    private $request;
    public function __construct(GroupInterface $group, Request $request)
    {
        $this->group = $group;
        $this->request = $request

Comment (xabbuh) [inline Resources/doc/index.rst]: @gruberjuliannama What do you mean?

Comment (gruberjuliannama) [inline Resources/doc/index.rst]: Milestone.."

File: Resources/doc/index.rst (hunk 1 above)
Comment (Pierstoval) [inline Resources/doc/index.rst]: Yep, juste saying `2.3+ and 3.0+`  is enough

**Comment by XWB** [comment]
Closing this PR as it is incorrect.

**Pull request closed without merging**